paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -59,10 +59,14 @@ class PaparazziPlugin : Plugin<Project> {
       }
     }
 
-    project.plugins.withId("com.android.library") { setupPaparazzi(project) }
+    val extension = project.extensions.create("paparazzi", PaparazziPluginExtension::class.java)
+    extension.snapshotDirectory.convention(project.layout.projectDirectory.dir("src/test/snapshots"))
```

Comment:
This'll probably need an integration test, to verify it can be overridden and it actually works in the new folder.